src/account.ts

```diff
@@ -0,0 +1,20 @@
+import { KeyringPair } from "@polkadot/keyring/types";
+
+class Account {
+  keyRingPair: KeyringPair;
+
+  constructor(keyRingPair: KeyringPair) {
```

Comment:
I'm curious, this key ring is what we would be accepting to make transactions on behalf of the user, instead of like requesting the wallet to sign the transaction?

Reply:
@yurixander  In this case, I'm letting the users create `KeyRingPair` on their own because this is an SDK, we are not interacting with wallet extensions on the browser. My current approach is to create a **class** `Account` that will perform tx to make it easier for the developer not to input the `KeyRingPair` every time they try to call the function. This can apply to the `webb-dapp` repo too, we can create a class with RPC endpoint so each time we call a function we do not need to put in the endpoint on every function. Related to this [PR](https://github.com/webb-tools/webb-dapp/pull/2102)

Reply:
This PR needs more updates, will re-request you again once it's ready @yurixander